bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/user-actions/component.jsx

```diff
@@ -82,6 +86,7 @@ const UserActions = (props) => {
     const toggleDisableCam = () => {
       if (!isCameraDisabled) {
         Session.set('disabledCams', [...disabledCams, cameraId]);
+        notify(intl.formatMessage(intlMessages.disableWarning), 'level', 'warning');
```

Comment:
![self](https://github.com/bigbluebutton/bigbluebutton/assets/3728706/7868a27a-b5c4-4e09-b0f9-2e7019f2b58f)


```suggestion
        notify(intl.formatMessage(intlMessages.disableWarning), 'info', 'warning');
```